myapp/src/day2/Question11/Cart.jsx

```diff
@@ -0,0 +1,37 @@
+import { useContext, useEffect, useState } from 'react'
+import { MyContext } from './App.jsx'
+
+const Cart = () => {
+  const { cart, setCart } = useContext(MyContext)
+  const [total, setTotal] = useState(0)
+  useEffect(() => {
+    const totalPrices = cart.reduce((acc, product) => acc + product.price, 0)
+    setTotal(totalPrices)
+  }, [cart])
+  return (
+    <div className="cart">
+      <h1>
+        Cart <span>{cart.length}</span>
+      </h1>
+      <p>Total Price: $ {total}</p>
+      <div>
+        {cart.map((product, index) => (
```

Comment:
card add and remove not working as expected

Reply:
total price and number of items in cart functionality is working now

Reply:
done: remove the unnecessary iteration